lib/service/service.go

```diff
@@ -885,28 +888,26 @@ func initExternalLog(auditConfig services.AuditConfig) (events.IAuditLog, error)
 				uri.Scheme, dynamo.GetName(), teleport.SchemeFile)
 		}
 	}
-	// only file external loggers are prohibited (they are not supposed
-	// to be used on their own, only in combo with external loggers)
-	// they also don't implement certain features, so they are going
-	// to be inefficient
-	switch len(loggers) {
-	case 0:
-		return nil, trace.NotFound("no external log is defined")
-	case 1:
-		if !hasNonFileLog {
-			return nil, trace.BadParameter(
-				"file:// or stdout:// log can not be used on it's own, " +
-					"can be only used in combination with external session logs, e.g. dynamodb://")
-		}
-		return loggers[0], nil
-	default:
-		if !hasNonFileLog {
-			return nil, trace.BadParameter(
-				"file:// or stdout:// log can not be used on it's own, " +
-					"can be only used in combination with external session logs, e.g. dynamodb://")
-		}
+
+	if len(loggers) < 1 {
+		return nil, nil
+	}
+
+	if !hasNonFileLog {
```

Comment:
I think we'll have customers running Teleport in environments where storing sessions locally is acceptable, I'd remove this warning.